desktop/cmp/filechooser/FileChooserModel.ts

```diff
@@ -113,19 +173,47 @@ export class FileChooserModel extends HoistModel {
     }
 
     @action
-    onDrop(accepted, rejected, enableMulti) {
-        if (!isEmpty(accepted)) {
-            if (!enableMulti) {
-                this.setSingleFile(accepted[0]);
-            } else {
-                this.addFiles(accepted);
-            }
+    onDrop(accepted: File[], rejected: FileRejection[]) {
+        if (accepted?.length > 1 && !this.enableMulti) {
+            accepted = [accepted[0]];
         }
-        this.lastRejectedCount = rejected.length;
+        this.lastAccepted = accepted;
+        this.lastRejected = rejected;
+        this.addFiles(accepted);
     }
 
-    onFilesChange(files) {
+    onFilesChange(files: File[]) {
         const fileData = files.map(file => ({name: file.name, size: file.size}));
         this.gridModel.loadData(fileData);
     }
+
+    //----------------
+    // Implementation
+    //----------------
+    private defaultTargetDisplay = () => {
+        return fragment(
+            p('Drag and drop your files here, or click to browse.'),
+            p({
+                items: [
+                    'Note that this example is configured to accept only ',
```

Comment:
This text sounds more appropriate for the toolbox examples then the default behavior in hoist.

Reply:
Agreed and changed - left the defaultTargetDisplay() returning the string in the first paragraph element as that was the default targetText before.